### ISSUE TITLE
datavec-data-image + CUDA: JVM crash on TestNativeImageLoader.testConvertPix

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/deeplearning4j#8910]
Fails locally for me - passes on CI.
Originally I thought it wasn't being run on CI due to this: https://github.com/eclipse/deeplearning4j/issues/8881#issuecomment-623247106
However it is reproducible without fixing that issue.
Also not introduced by my NHWC PR recently - still fails as of commit hash 4dbdaca967dd9393c0de7df5f8edead74e0dbb7a (`Add support for registering custom loss functions for Keras import (#427)`)

It's reproducible by running datavec-data-image tests (intellij or command line - `mvn clean test -Ptest-nd4j-cuda-10.2 -Ptestresources`):

After:
```
o.n.c.t.BaseND4JTest - TestNativeImageLoader.testConvertPix
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x0000025836205fc5, pid=30104, tid=0x0000000000008e54
#
# JRE version: OpenJDK Runtime Environment (8.0_242-b08) (build 1.8.0_242-b08)
# Java VM: OpenJDK 64-Bit Server VM (25.242-b08 mixed mode windows-amd64 compressed oops)
# Problematic frame:
# J 1615 C2 org.bytedeco.javacpp.indexer.UByteRawIndexer.get(JJJ)I (29 bytes) @ 0x0000025836205fc5 [0x0000025836205f60+0x65]
#
# Failed to write core dump. Minidumps are not enabled by default on client versions of Windows
#
# An error report file with more information is saved as:
# c:\DL4J\git\deeplearning4j\datavec\datavec-data\datavec-data-image\hs_err_pid30104.log
Compiled method (c1)   83235 3073       3       org.datavec.image.loader.NativeImageLoader::fillNDArray (1397 bytes)
 total in heap  [0x00000258362b7990,0x00000258362c47b8] = 52776
 relocation     [0x00000258362b7ab0,0x00000258362b8248] = 1944
 main code      [0x00000258362b8260,0x00000258362bf8a0] = 30272
 stub code      [0x00000258362bf8a0,0x00000258362bfca0] = 1024
 oops           [0x00000258362bfca0,0x00000258362bfcd8] = 56
 metadata       [0x00000258362bfcd8,0x00000258362bfdb0] = 216
 scopes data    [0x00000258362bfdb0,0x00000258362c37f0] = 14912
 scopes pcs     [0x00000258362c37f0,0x00000258362c4650] = 3680
 dependencies   [0x00000258362c4650,0x00000258362c4678] = 40
 nul chk table  [0x00000258362c4678,0x00000258362c47b8] = 320
#
# If you would like to submit a bug report, please visit:
#
#

Results :

Tests run: 15, Failures: 0, Errors: 0, Skipped: 1

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:34 min
[INFO] Finished at: 2020-05-04T14:48:33+10:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project datavec-data-image: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test failed: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was cmd.exe /X /C ""C:\Program Files\AdoptOpenJDK\jdk-8.0.242.08-hotspot\jre\bin\java" -Ddtype=float -Dfile.encoding=UTF-8 org.apache.maven.surefire.booter.ForkedBooter c:\DL4J\git\deeplearning4j\datavec\datavec-data\datavec-data-image\target\surefire\surefire650159490836136796tmp c:\DL4J\git\deeplearning4j\datavec\datavec-data\datavec-data-image\target\surefire\surefire_03224998358035242936tmp"
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

JVM crash log: https://gist.github.com/AlexDBlack/b6071c532c1318a363d999e0f9b529ed

**Comment by raver119** [comment]
This is duplicate issue.

**Comment by raver119** [comment]
https://github.com/eclipse/deeplearning4j/issues/8785

**Comment by agibsonccc** [comment]
Closing — references a removed/rewritten subsystem (DataVec, NLP/Word2Vec, Model Zoo, Spark, UI, or Arbiter). These have been fundamentally rewritten or removed in the upcoming release. Please open a new issue if you experience similar problems with the new version.
